hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -299,4 +300,59 @@ public static void validatePreviousSnapshotId(SnapshotInfo snapshotInfo,
           OMException.ResultCodes.INVALID_REQUEST);
     }
   }
+
+  /**
+   * Compares the block location info of 2 key info.
+   * @return true if block locations are same else false.
+   */
+  public static boolean isBlockLocationInfoSame(OmKeyInfo prevKeyInfo,
```

Comment:
this is a relatively complex helper method. A test for this helper for various corner cases would be ideal.

Reply:
Since this change doesn't pertain to this PR as I have simply moved this function to SnapshotUtils I can create a follow up task for adding a test case for this.